.travis.yml

```diff
@@ -50,7 +50,12 @@ before_install:
     fi
   - if [[ "$TRAVIS_OS_NAME" == "linux" ]]; then sudo paddle/scripts/travis/before_install.linux.sh; fi
   - if [[ "$TRAVIS_OS_NAME" == "osx" ]]; then paddle/scripts/travis/before_install.osx.sh; fi
-  - pip install wheel protobuf 'sphinx==1.4.9' breathe recommonmark virtualenv numpy
+  - pip install wheel  'sphinx==1.4.9' breathe recommonmark virtualenv numpy
+  # In master branch, protobuf 2.5 is used when compiling Paddle. And google change the
```

Comment:
我一般写邮件和谐代码注释的时候会用 grammerly.com 做一下英语语法检查。

建议修改如下

> PaddlePaddle's master branch uses protobuf 2.5. And Google changed the float number serialization code in protobuf 3.x. So, if we use the most recent version of protobuf, PaddlePaddle unit tests would fail because we compare protobuf serialization results using the `diff` command.

另一个简单粗暴的规则是：不要用被动语态。